src/platform_impl/macos/window_delegate.rs

```diff
@@ -172,9 +172,13 @@ declare_class!(
             let filenames = pb.propertyListForType(unsafe { NSFilenamesPboardType });
             let filenames: Id<NSArray<NSString>, Shared> = unsafe { Id::cast(filenames) };
 
+            let dl: NSPoint = unsafe { msg_send![sender, draggingLocation] };
+            let scale_factor = self.window.scale_factor();
+            let position = LogicalPosition::<f64>::from((dl.x, dl.y)).to_physical(scale_factor);
```

Comment:
We're missing a conversion to `winit`'s coordinate system, which has origin in the top-left (e.g. use `util::bottom_left_to_top_left`).

Same below.